### DISCUSSION TITLE
menu for different languages

### DISCUSSION THREAD

**Discussion by sweetkatrin** [thegeeklab/hugo-geekdoc#211] [Q&A]
Hi I am doing documentation using your theme. There was a problem with different languages. When generating a menu, the menu structure is not displayed in the Russian version, while in the English version everything works fine. What could be the problem?

ru version
![image](https://user-images.githubusercontent.com/36506227/134671054-36d6e927-f2c7-4864-8a61-62ecb4cfc63b.png)

eng version 
![image](https://user-images.githubusercontent.com/36506227/134671219-ffbac83d-1f50-420f-894c-e3eb738f59ee.png)

file tree
![image](https://user-images.githubusercontent.com/36506227/134671282-f90b854e-411e-49be-9171-205d2d7eee66.png)

**Comment by xoxys** [comment]
Hi, hmm not sure right now. How do you handle/build the different language versions? Keep in mind that the theme does not support Hugos [multilingual](https://gohugo.io/content-management/multilingual/) mode right now.
